infra/main.bicep

```diff
@@ -85,14 +106,38 @@ module proxy 'proxy.bicep' = {
     containerAppsEnvironmentName: containerApps.outputs.environmentName
     containerRegistryName: containerApps.outputs.registryName
     exists: proxyAppExists
-    postgresServer: postgresServer.outputs.DOMAIN_NAME
+    postgresServer: '${prefix}-postgresql.postgres.database.azure.com'
     postgresDatabase: postgresDatabaseName
-    postgresUser: postgresAdminUser
-    postgresPassword: postgresAdminPassword
+    postgresUser: '${prefix}-proxy'
```

Comment:
This should either use an output (see other comments) or a variable, to reduce the potential for mistakes.